oauth2/oauth2-jose/src/main/java/org/springframework/security/oauth2/jwt/validation/FailedValidationJwtException.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2002-2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.security.oauth2.jwt.validation;
+
+import java.util.Collection;
+
+import org.springframework.security.oauth2.core.OAuth2Error;
+import org.springframework.security.oauth2.core.valiation.OAuth2TokenValidatorResult;
+import org.springframework.security.oauth2.jwt.JwtException;
+import org.springframework.util.Assert;
+
+/**
+ * @author Josh Cummings
+ * @since 5.1
+ */
+public class FailedValidationJwtException extends JwtException {
```

Comment:
I think `Failed` is redundant given that it's an `Exception`. What do you think about `JwtValidationException`?

Reply:
Yeah, you're right. Failed is redundant.